commands/faas_test.go

```diff
@@ -0,0 +1,52 @@
+package commands
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestLoadsDefaultYAMLWhenPresent(t *testing.T) {
+	rs := resetState()
+	defer rs()
+	os.Chdir("./testdata")
```

Comment:
I think this works, but wondered if we could fake the result of `os.Stat`?

Reply:
In this usage I effectively "faked" OS env:

https://github.com/openfaas/faas/blob/master/watchdog/config_test.go#L34